ccmlib/scylla_cluster.py

```diff
@@ -100,13 +100,15 @@ def __update_pids(self, started):
             node._update_pid(p)
 
     def start_nodes(self, nodes=None, no_wait=False, verbose=False, wait_for_binary_proto=None,
-              wait_other_notice=None, jvm_args=None, profile_options=None,
+              wait_other_notice=None, wait_normal_token_owner=None, jvm_args=None, profile_options=None,
```

Comment:
To be honest, I don't understand why anybody will want to set this wait_normal_token_owner thing - in what situation would you want not to wait?
And I don't understand how anyone would ever understand what it means without a comment explaining it.
Also, why is the default "None" and not True or False??

Reply:
I see below that you have in start_node() a comment about this option. My other questions on it still stand.

Reply:
> To be honest, I don't understand why anybody will want to set this wait_normal_token_owner thing - in what situation would you want not to wait?

It's for backward compatibility purposes in case this change breaks existing dtests that rely on the previous timing (like they need to inject some even after they node is noticed as UP and before its considered a normal token owner.

This was the case I fixed in https://github.com/scylladb/scylla-dtest/pull/3716.
We can follow these lines also in other test cases and just pass `wait_other_notice=False`, but I think this option would provide a faster path that will just fallback to the existing behavior.

> And I don't understand how anyone would ever understand what it means without a comment explaining it.

I'll spread the doc comment around more broadly.

> Also, why is the default "None" and not True or False??

Like the oter args, `None` let's the test know that it is unset, and then it is calculated in the function body based on the state of other parameters.  It's hard to achieve that with literal default initialization.

Reply:
I remembered the None thing is a Python convention, you're right.

I'm not a fan of dtest's approach of having convenience functions that take 10 parameters that nobody understand instead of just "doing the right thing" but I agree that functionally, the extra parameter doesn't hurt. So if the dtest developers are fine with it, then so be it.